### PR TITLE
Steddy us 1

### LINKED ISSUE #11: 19. Admin Dashboard
As an admin,
When I visit the admin dashboard (/admin)
Then I see a header indicating that I am on the admin dashboard

### PR DESCRIPTION
**Pull request by Steddy1Love**
Close #11  and #12

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
-  end
-
   resources :merchants, only: [:show] do
     member { get "dashboard"}
     resources :items, only: [:index]
     resources :invoices, only: [:index]
   end
-end
+
+  get '/admin', to: 'admin#index'
```
Comment (Steddy1Love) [inline config/routes.rb:15]: Hand rolled this route for now. This can be something we bring up during our check-in

**Review by Steddy1Love — COMMENTED** [review]
Left a comment on something I want to bring up during check in.

File: config/routes.rb (hunk 1 above)
Comment (NicoShanstrom) [reply config/routes.rb:15]: good as long as it works for now we can refactor it later when necessary

**Review by NicoShanstrom — COMMENTED** [review]
looks good. will have to play with that handrolled route later though.

**Pull request closed without merging**